tests/e2e/provisioning/test/e2e_provisioning_test.go

```diff
@@ -0,0 +1,25 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_E2E_Provisioning(t *testing.T) {
+	ts := newTestSuite(t)
+	operationID, err := ts.brokerClient.ProvisionRuntime()
+	require.NoError(t, err)
+	defer ts.TearDown()
+
+	err = ts.brokerClient.AwaitProvisioningSucceeded(operationID)
```

Comment:
**nit:** I like API where I can explicit set the timeout for such "await" and I do not need to go into impl details. As a result it is more readable for me.